search/src/test/scala/weco/api/search/services/WorksServiceTest.scala

```diff
@@ -413,49 +221,34 @@ class WorksServiceTest
     }
 
     it("aggregates formats") {
-      withLocalWorksIndex { _ =>
-        val work1 = indexedWork().format(Books)
-        val work2 = indexedWork().format(Books)
-        val work3 = indexedWork().format(Audio)
-        val work4 = indexedWork().format(ArchivesAndManuscripts)
-
-        val worksSearchOptions =
-          createWorksSearchOptionsWith(
-            aggregations = List(WorkAggregationRequest.Format)
-          )
-
-        val expectedAggregations = WorkAggregations(
-          Some(
-            Aggregation(
-              List(
-                AggregationBucket(data = Books, count = 2),
-                AggregationBucket(data = ArchivesAndManuscripts, count = 1),
-                AggregationBucket(data = Audio, count = 1)
-              )
+      val expectedAggregations = WorkAggregations(
+        Some(
+          Aggregation(
+            List(
+              AggregationBucket(data = Format.Books, count = 4),
```

Comment:
Should this be `count=worksFormatBooks.length` etc.

Reply:
Ooh, yes, good suggestion.